cmd/release/cmd/errors.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import "fmt"
+
+type VersionNotFoundError struct {
+	Version string
+}
+
+func (e *VersionNotFoundError) Error() string {
+	return fmt.Sprintf("verify your config file, version not found: %s", e.Version)
```

Comment:
Concatenate these strings. Always avoid xPrintf functions for strings unless the string has a complex format.

Reply:
Done